src/uikit/DataTable/Pagination/CustomPagination.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import classnames from 'classnames';
+import { range } from 'lodash';
+import ReactTablePagination from 'react-table/lib/pagination';
+
+import { Pagination } from './styles';
+
+export default class CustomPagination extends ReactTablePagination {
```

Comment:
For sure. Using a class here because it was essentially a copy paste of an Arranger component.
For writing new components I'd definitely steer more towards `react-component-component` or `recompose`